### ISSUE TITLE
"Encountered errors during upload" but no further information

### ISSUE THREAD

**Issue by Porges** [DavidMStraub/gramps-web-sync#31]
Hi, I'm trying to sync media objects but some of them fail (consistently) and there's no output in the console.

----

Gramps version: AIO64-5.2.3-r1-aa03f5a
Gramps Web version:
```
Gramps 5.2.2
Gramps Web API 2.3.1
Gramps Web Frontend 24.7.0
Gramps QL 0.3.0
locale: en
multi-tree: false
task queue: true
```

Gramps Web Sync version: 1.1.1

----

Is there anything I can do to show what's wrong? 

Thanks

**Comment by DavidMStraub** [comment]
Hi, could it be that the file has the wrong checksum in the Gramps DB? This often happens for editable files like Office files. See also https://www.grampsweb.org/user-guide/sync/

**Comment by DavidMStraub** [comment]
[Version 1.2.0](https://github.com/gramps-project/addons-source/pull/626) which will be released soon might fix this error or, if not, introduces much better debug logging. Since I will archive this repo in favour of the official addons-source repo, I'll close this issue, but please let me know if the problem is solved one 1.2.0 is released.
